t/api/04-jobs.t

```diff
@@ -127,4 +127,23 @@ $post->content_is('OK');
 isnt -e $rp, undef, "logs exist after";
 is(calculate_file_md5($rp), "feeebd34e507d3a1641c774da135be77", "md5sum matches");
 
+
+$rp = "t/data/openqa/factory/hdd/hdd_image.qcow2";
+unlink($rp);
+$post = $t->post_ok('/api/v1/jobs/99963/artefact' => form => {file => {file => $filename, filename => 'hdd_image.qcow2'}, asset => 'public'})->status_is(200);
+$post->content_is('OK');
+isnt -e $rp, undef, "asset exist after";
```

Comment:
This is such a strange construct. How about ok(-e $rp, 'asset exist after submitting');?